.circleci/config.yml

```diff
@@ -668,21 +668,19 @@ jobs:
   publish-crates:
     parameters:
       path:
-        description: Crate to publish
+        description: Path to crate to publish
         type: string
     executor: docker-rust
-    resource_class: medium
     steps:
       - checkout
       - run:
-          name: Crate publishing in order
-          command: |
-            cargo publish --manifest-path << parameters.path >>/Cargo.toml
+          name: Publish crate
+          # Don't build the crate, we know that the crate should build based on other workflows
+          command: cargo publish --no-verify --manifest-path << parameters.path >>/Cargo.toml
```

Comment:
I know this should be safe, but it still makes me nervous. :sweat_smile: I can't think of any edge cases where this might be a problem at the moment, so I think it's fine.

Reply:
If you want faster releases, we need to release faster :blush:

Reply:
Indeed 😂